peps/pep-0756.rst

```diff
@@ -150,18 +152,40 @@ flags.
 
 Note that future versions of Python may introduce additional formats.
 
+By default, if the ``PyUnicode_EXPORT_COPY`` flag is not set in
+*flags*, no memory is copied and no conversion is done.
+
+If the ``PyUnicode_EXPORT_COPY`` flag is set in *flags*, the function
+can copy memory to provide the requested format and convert from a format
+to another.
+
+The ``PyUnicode_EXPORT_COPY`` flag is needed to export to
+``PyUnicode_FORMAT_UTF8`` a string containing surrogate characters.
+
+Available flags:
+
+=========================  ========  ===================================
+Flag                       Value     Description
+=========================  ========  ===================================
+``PyUnicode_EXPORT_COPY``  ``0x01``  Allow memory copies and conversions
```

Comment:
I would say `PyUnicode_EXPORT_ALLOW_COPY`, otherwise this might imply that a copy is always done.